src/functions-and-arrays.js

```diff
@@ -1,19 +1,65 @@
 // Iteration #1: Find the maximum
+function maxOfTwoNumbers(number1, number2){
+  if (number1 < number2) {
+    return number2;
+    
+  } else {
+    return number1;
+  }
+}
+
+console.log(maxOfTwoNumbers(3,4));
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
+function findLongestWord(array){
+  var longestWord = 5;
+for (i=0; i<array.length; i++){
```

Comment:
The logic was good but you were returning a number and the exercise asks you for a word, you can fix it like this:

var longestWord = "";
for (i=0; i<array.length; i++){
  if (array[i].length > longestWord.length) {
    longestWord = array[i];

  }
}
 return longestWord;